<Type Name="XamlSchemaException" FullName="System.Xaml.XamlSchemaException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="434fd61bb9b35ec86aa408dad04779d301def3f7" /><Meta Name="ms.sourcegitcommit" Value="1b924db57b3a1cf768d98c21f9b988d6966a0f2b" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="ja-JP" /><Meta Name="ms.lasthandoff" Value="03/06/2020" /><Meta Name="ms.locfileid" Value="78731816" /></Metadata><TypeSignature Language="C#" Value="public class XamlSchemaException : System.Xaml.XamlException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit XamlSchemaException extends System.Xaml.XamlException" FrameworkAlternate="netcore-3.0;netcore-3.1" />
  <TypeSignature Language="DocId" Value="T:System.Xaml.XamlSchemaException" />
  <TypeSignature Language="VB.NET" Value="Public Class XamlSchemaException&#xA;Inherits XamlException" />
  <TypeSignature Language="C++ CLI" Value="public ref class XamlSchemaException : System::Xaml::XamlException" />
  <TypeSignature Language="F#" Value="type XamlSchemaException = class&#xA;    inherit XamlException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit XamlSchemaException extends System.Xaml.XamlException" FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
  <AssemblyInfo>
    <AssemblyName>System.Xaml</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Xaml.XamlException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="e8b98-101">XAML のバインディング システムまたは別のスキーマ表現システムがスキーマ コンテキストに対して例外を報告した場合にスローされる例外です。</span><span class="sxs-lookup"><span data-stu-id="e8b98-101">The exception that is thrown when a binding system or another schema representation system for XAML reports an exception to the schema context.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="e8b98-102"><xref:System.Xaml.XamlSchemaException> は、.NET Framework XAML サービス (<xref:System.Xaml.XamlXmlReader>、<xref:System.Xaml.XamlObjectWriter>、または <xref:System.Xaml.XamlServices>) のスキーマコンテキストを提供する CLR バインディングサポートクラスのいくつかのケースによってスローされます。</span><span class="sxs-lookup"><span data-stu-id="e8b98-102">The <xref:System.Xaml.XamlSchemaException> is thrown by several cases in the CLR binding support classes that provide the schema context for .NET Framework XAML Services (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, or <xref:System.Xaml.XamlServices>).</span></span> <span data-ttu-id="e8b98-103">通常、<xref:System.Xaml.XamlSchemaException> は、<xref:System.Xaml.XamlObjectWriterException>の <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>-true プロパティなど、他の例外の内部例外として渡されます。そのため、特化されたオブジェクトライターなどのコンポーネントを実装する場合を除き、<xref:System.Xaml.XamlSchemaException> を直接キャッチすることは珍しくありません。</span><span class="sxs-lookup"><span data-stu-id="e8b98-103">A <xref:System.Xaml.XamlSchemaException> is typically passed as the inner exception for other exceptions, such as the <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>-true property for a <xref:System.Xaml.XamlObjectWriterException>, so it should be uncommon to catch a <xref:System.Xaml.XamlSchemaException> directly unless you are implementing a component such as a specialized object writer.</span></span>  
  
 ]]></format>
    </remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="e8b98-104"><see cref="T:System.Xaml.XamlSchemaException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-104">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="e8b98-105">エラーを説明するシステム提供のメッセージを使用して、<see cref="T:System.Xaml.XamlSchemaException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-105">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a system-supplied message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="e8b98-106">例外を説明するメッセージ。</span><span class="sxs-lookup"><span data-stu-id="e8b98-106">The message that describes the exception.</span></span> <span data-ttu-id="e8b98-107">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="e8b98-107">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <summary><span data-ttu-id="e8b98-108">エラーを説明する指定したメッセージを使用して、<see cref="T:System.Xaml.XamlSchemaException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-108">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; XamlSchemaException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="e8b98-109">シリアル化されたオブジェクト データを保持するオブジェクト。</span><span class="sxs-lookup"><span data-stu-id="e8b98-109">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="e8b98-110">転送元または転送先に関するコンテキスト情報。</span><span class="sxs-lookup"><span data-stu-id="e8b98-110">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="e8b98-111">シリアル化したデータを使用して、<see cref="T:System.Xaml.XamlSchemaException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-111">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with serialized data.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string * Exception -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="e8b98-112">例外を説明するメッセージ。</span><span class="sxs-lookup"><span data-stu-id="e8b98-112">The message that describes the exception.</span></span> <span data-ttu-id="e8b98-113">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="e8b98-113">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <param name="innerException"><span data-ttu-id="e8b98-114">現在の例外の原因となった例外。</span><span class="sxs-lookup"><span data-stu-id="e8b98-114">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="e8b98-115"><paramref name="innerException" /> パラメーターが <see langword="null" /> でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-115">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="e8b98-116">指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を使用して、<see cref="T:System.Xaml.XamlSchemaException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-116">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="e8b98-117">前の例外の直接の結果としてスローされる例外については、<xref:System.Exception.InnerException%2A> プロパティに、前の例外への参照を含める必要があります。</span><span class="sxs-lookup"><span data-stu-id="e8b98-117">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="e8b98-118"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。`null` プロパティによって内部例外値がコンストラクターに渡されなかった場合は、<xref:System.Exception.InnerException%2A> を返します。</span><span class="sxs-lookup"><span data-stu-id="e8b98-118">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>
